src/utils.c

```diff
@@ -265,6 +234,27 @@ SEXP s3_find_method(const char* generic, SEXP x) {
 }
 
 
+// Initialised at load time
+SEXP compact_seq_attrib = NULL;
+
+// Returns a compact sequence that `vec_slice()` understands
+SEXP compact_seq(R_len_t from, R_len_t to) {
```

Comment:
Not sure where this is used, but should it check that `to` > `from`? (or return empty vector)